backend/src/models/statusUpdate.js

```diff
@@ -0,0 +1,66 @@
+const { Model } = require('objection')
+const tableStatusUpdates = 'statusUpdates'
+
+class StatusUpdate extends Model {
+  static get tableName () {
+    return tableStatusUpdates
+  }
+
+  static get idColumn () {
+    return ['StatusUpdateID']
+  }
+
+  static get relationMappings () {
+    const Student = require('./student')
+    const Status = require('./status')
+
+    return {
+      student: {
+        relation: Model.HasOneRelation,
```

Comment:
```suggestion
        relation: Model.BelongsToOneRelation,
```
Here, the source model `StatusUpdate` has the foreign key, while the related model `Student` has the primary key `StudentID`. https://vincit.github.io/objection.js/guide/relations.html#examples